docs/self-managed/deployment/helm/configure/authentication-and-authorization/generic-oidc-provider.md

```diff
@@ -1,146 +1,784 @@
+
+- `issuer` and `publicIssuerUrl`: `https://login.example.com`
+- `issuerBackendUrl`: `http://oidc-provider.namespace.svc.cluster.local:8080`
+  :::
+
+### Configure Management Identity
+
+Add configuration for Management Identity:
+
+```yaml
+global:
+  identity:
+    auth:
+      identity:
+        clientId: <identity-client-id>
+        audience: <identity-audience>
+        redirectUrl: <identity-base-url>
+        secret:
+          existingSecret: oidc-credentials
+          existingSecretKey: identity-client-secret
+        initialClaimName: <user-claim-name>
+        initialClaimValue: <admin-user-claim-value>
+
+identity:
+  enabled: true
+  contextPath: "/identity" # Optional: adjust if using custom context path
+
+identityPostgresql:
+  enabled: true
+  auth:
+    existingSecret: camunda-credentials
+    secretKeys:
+      adminPasswordKey: identity-postgresql-admin-password
+      userPasswordKey: identity-postgresql-user-password
+```
+
+**Identity-specific parameters:**
+
+| Parameter           | Description                                  | How to Determine                                                                                |
+| ------------------- | -------------------------------------------- | ----------------------------------------------------------------------------------------------- |
+| `clientId`          | Client ID from your OIDC provider            | From your Identity client configuration                                                         |
+| `audience`          | Expected audience in access tokens           | From token inspection (see [Discover provider configuration](#discover-provider-configuration)) |
+| `redirectUrl`       | Full URL where Identity is accessible        | `http://localhost:8084` (local) or `https://your-domain.com/identity` (Ingress)                 |
+| `initialClaimName`  | Claim that identifies the initial admin user | `email`, `sub`, or another user claim from token inspection                                     |
+| `initialClaimValue` | Value granting initial admin access          | Your admin user's value for the specified claim (e.g., `admin@example.com`)                     |
+
+:::danger Initial claim cannot be changed
+The `initialClaimName` and `initialClaimValue` parameters are used only during the first startup to grant initial admin access. Once Management Identity has started, these values are stored in the database and cannot be changed via Helm values.
+:::
+
+### Configure Orchestration Cluster
+
+Add configuration for the Orchestration Cluster (Zeebe, Operate, Tasklist):
+
+```yaml
+global:
+  identity:
+    auth:
+      orchestration:
+        clientId: <orchestration-client-id>
+        audience: <orchestration-audience>
+        secret:
+          existingSecret: oidc-credentials
+          existingSecretKey: orchestration-client-secret
```

Comment:
This is redundant with the section in `orchestration`. Is there a particular reason why you have it in both places? Otherwise I suggest in line with our ambition to limit the use of global values to only have the `orchestration` section. That worked for me when I wrote the Entra/external Keycloak guides.